var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -14,6 +14,7 @@ class Brayns(CMakePackage):
 
     version('develop', submodules=True)
     version('0.8.0', tag='0.8.0', submodules=True, preferred=True)
+    version('1.0.0', tag='1.0.0', submodules=True)
```

Comment:
If you are setting 0.8 as preferred, we should be able to remove '^brayns@0.8.0` above from the spec (simpler should be better).